pkg/services/libraryelements/database.go

```diff
@@ -202,14 +203,46 @@ func (l *LibraryElementService) createLibraryElement(c context.Context, signedIn
 		},
 	}
 
+	if l.features.IsEnabled(featuremgmt.FlagLibraryPanelRBAC) {
+		if err := l.updatePermissions(c, &dto, signedInUser); err != nil {
+			l.log.Error(err.Error())
+		}
+	}
+
 	return dto, err
 }
 
+func (l *LibraryElementService) updatePermissions(c context.Context, model *model.LibraryElementDTO, signedInUser identity.Requester) error {
```

Comment:
This would be needed if we want to allow setting permissions on an individual library panel level. If we're fine with only granting permissions on a folder level (either you can or can't CRUD library elements in a given folder), then this would not be necessary. What do you think?

Reply:
Not actually sure quite how fine-grained we need things to be. I'll try and get an answer for you.

Reply:
Got some clarification and we do actually want permissions to be at the individual library panel level